types/supercluster/supercluster-tests.ts

```diff
@@ -1,23 +1,66 @@
 import supercluster, { Point } from 'supercluster';
 
+interface TestMapProperties {
```

Comment:
No longer needed since we are using `any` to define the properties.

Reply:
yea will change the test's interface declaration